src/components/Display.jsx

```diff
@@ -0,0 +1,11 @@
+const Display = () => {
+    return (
+        <select>
+            <option>Votes</option>
+            <option>Comments</option>
+            <option>Date</option>
+        </select>
+    )
+}
+
+export default Display;
```

Comment:
no need to add this code yet - this isn't part of the ticket!

Reply:
Ok I'll remove it, thank you!